src/schema/level.ts

```diff
@@ -0,0 +1,17 @@
+export enum LogLevel {
```

Comment:
what is the benefit of setting this as an enum instead of the const for the LogLevel? would you not avoid the issues coming from the Object.entries?

Reply:
Yeah, I was umming and arring about that one. On the one hand an enum is the idiomatic way to do something like this (a 1:1 mapping of human-readable thing to unique, integer ID) so I started, then I discovered this typescript annoyance. In fairness, it doesn't actually change the amount of code very much, in this PR we have

```typescript
const entries = Object.entries(LogLevel);
return entries.slice(entries.length / 2) as [string, number][];
```

If we stuck with the dictionary from before we would need

```typescript
return Object.entries(LogLevel);
```

Which is slightly less code, but not a _huge_ amount, and the enum is more readable than the dictionary so that feels like an okay trade off?

Reply:
Yes that makes sense!